### DISCUSSION TITLE
Managing required code for optional modules in Starter Kits

### DISCUSSION THREAD

**Discussion by sstottelaar** [statamic/cms#10785] [Q&A] [answered]
First off, love the new Starter Kit Modules feature! That brings me to a question about handling the required code for these packages. For example, if I add `statamic/seo-pro` as a module, it needs specific code in the head tag to function properly. Since it’s optional, is there a way to check if the module is installed? I guess adding it to the README is a simple enough solution, but hey, maybe there’s a cleaner way to handle it.

Thanks!

**Comment by duncanmcclean** [comment, marked as answer]
I'm assuming you're asking about checking it in your templates, if so, you can use the [`installed`](https://statamic.dev/tags/installed) tag for that:

```antlers
{{ if {installed:statamic/seo-pro} }}
    {{ seo_pro:meta }}
{{ /if }}
```
